app/views/main/_navbar.html.erb

```diff
@@ -12,6 +12,7 @@
 
     <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
       <ul class="nav navbar-nav navbar-right">
+      <%= link_to 'My Profile', profile_path, class: 'btn btn-primary' %>
```

Comment:
shouldn't be this activated when user is instructor

Reply:
don't we need profile for trainee as well?